### DISCUSSION TITLE
Duplicate multiple automations at once.

### DISCUSSION THREAD

**Discussion by niemyjski** [home-assistant/frontend#7812] [Automations / scripts / scenes]
I have two automations for every room, one to turn it off and one to turn it on. It would be awesome if I could select multiple automations from the grid and duplicate them.

**Comment by KTibow** [comment]
Just curious, why couldn't you use templating and the choose operator? Ideally you'd have two big automations.

**Reply by niemyjski** [reply]
I'm digging deeper into templating but my triggers/triggers between rooms are slightly different but there is some commonality so I find duplicating makes sure I have everything I need.

**Reply by KTibow** [reply]
> the grid

What do you mean by this?

**Reply by niemyjski** [reply]
The grid where you view all of your automations.

**Reply by KTibow** [reply]
For me it's more of a list. Did they update that?

**Reply by niemyjski** [reply]
It's always been a list for me. Would be nice to duplicate from there
